CONTRIBUTING.md

```diff
@@ -165,20 +216,14 @@ which will mean that the issue is automatically linked to the pull request in th
 
 
 
-#### Pull Request Review
+### Pull Request Review
 
 The project manager(s) will review the pull request for coherence with the specified feature or bug fix, and give feedback on code quality, user experience, documentation and git style.  Please respond to comments from the project managers with explanation, or further commits to your pull request in order to get merged in as quickly as possible.
 
-To maximize flexibility, you can add the project managers as collaborators to your projectscope fork in order to allow them to help you fix your pull request, but this is not required.
-
+To maximize flexibility, you can add the project managers as collaborators to your fork in order to allow them to help you fix your pull request, but this is not required.
```

Comment:
Adding managers as collaborators is not really necessary. Github  makes this process simple nowadays. Just ensure when creating a PR That you `Allow edits from maintainers`, and every contributors should be able to push changes to your branch to update the PR.

![screenshot 2016-11-12 16 50 26](https://cloud.githubusercontent.com/assets/4028374/20238630/8e0b66da-a8f8-11e6-82c2-cf24cbaf40b7.png)